python/tvm/meta_schedule/relay_integration.py

```diff
@@ -385,7 +401,9 @@ def compile_relay(
                 config=pass_config,
             ):
                 if backend == "graph":
-                    return relay.build(mod, target=target, params=params, executor=executor)
+                    return relay.build(
+                        mod, target=target, params=params, executor=executor, runtime=runtime
```

Comment:
Would this have any impact on non-cpp build? Asking because it seems after parameter normalizaion the runtime would be set to `relay.backend.Runtime("cpp")`.

Reply:
Again, I don't think it would affect other target usage, since the `Runtime("cpp")` is the default runtime in the `relay.build`. I'll wait for others to comment on this.